parlai/messenger/core/messenger_manager.py

```diff
@@ -266,9 +266,10 @@ def _on_new_message(self, message):
                 self.observe_message(
                     agent_id,
                     "Please wait while we pair you with another person. "
-                    "If you wish to return to the Overworld, click *EXIT*",
+                    "If you wish to exit, click *EXIT*.",
```

Comment:
Note - we should probably drop the quick reply and ask a user to manually type exit, as we've seen that people like just clicking quick replies without reading